### PR TITLE
Feature/additional vob data and c item export

### PR DESCRIPTION
**Pull request by JaXt0r**
To implement C_Items inside chests on CSharp side, I put exports for two main functions:
1. World --> Subtyped Vob elements. Currently created for Mobs and Zones. (Other elements are marked with a //TODO)
2. Vm --> Load C_Item element data (aligned with C_Npc loading)

Misc:
* The functions are tested on csharp interface already.
* linting is green

### CODE REVIEW

File: include/phoenix/cffi/World.h [hunk 1]
```diff
@@ -139,3 +231,62 @@ PXC_API PxVobVisualType pxVobGetVisualType(PxVob const* vob);
...
+// Vob - ZoneFarPlane
+PXC_API void
+pxWorldVobGetZoneFarPlane(PxVobZoneFarPlane* zoneFarPlane, float* vob_far_plane_z, float* inner_range_percentage);
+// Vob - ZoneFog
+PXC_API void pxWorldVobGetZoneFog(PxVobZoneFog* zoneFog,
+                                  float* range_center,
+                                  float* inner_range_percentage,
+                                  glm::u8vec4* color,
+                                  bool* fade_out_sky,
+                                  bool* override_color);
```
Comment (lmichaelis) [inline include/phoenix/cffi/World.h:292]: Please add a newline at the end of files.

File: src/Vm.cc [hunk 2]
```diff
@@ -421,3 +429,16 @@ char const* pxVmInstanceNpcGetName(PxVmInstance const* instance, uint32_t i) {
...
+	return RCC(phoenix::c_item, instance)->name.c_str();
+}
+
+char const* pxVmInstanceItemGetDescription(PxVmInstance const* instance) {
+	return RCC(phoenix::c_item, instance)->description.c_str();
+}
+
+char const* pxVmInstanceItemGetVisual(PxVmInstance const* instance) {
+	return RCC(phoenix::c_item, instance)->visual.c_str();
+}
```
Comment (lmichaelis) [inline src/Vm.cc:444]: Please add a newline at the end of files.

File: src/World.cc [hunk 3]
```diff
@@ -177,3 +177,148 @@ uint32_t pxVobGetChildCount(PxVob const* vob) {
...
+                          float* inner_range_percentage,
+                          glm::u8vec4* color,
+                          bool* fade_out_sky,
+                          bool* override_color) {
+	*range_center = zoneFog->range_center;
+	*inner_range_percentage = zoneFog->inner_range_percentage;
+	*color = zoneFog->color;
+	*fade_out_sky = zoneFog->fade_out_sky;
+	*override_color = zoneFog->override_color;
+}
```
Comment (lmichaelis) [inline src/World.cc:324]: Please add a newline at the end of files.

**Review by lmichaelis — CHANGES_REQUESTED** [review]
Thanks for the PR. I just have some minor issues with missing newlines at the end of some files.

**Comment by JaXt0r** [comment]
Somehow my new changes aren't reflected in this PR. I open a new one.

**Pull request closed without merging**